query/src/main/scala/filodb/query/exec/PlanDispatcher.scala

```diff
@@ -19,7 +20,7 @@ import filodb.query.QueryResponse
   */
 trait PlanDispatcher extends java.io.Serializable {
   def clusterName: String
-  def dispatch(plan: ExecPlan)
+  def dispatch(plan: ExecPlan, source: ChunkSource)
```

Comment:
Adding a `ChunkSource` here to pass to any `InProcessPlanDispatcher`s. These previously never dispatched leaf nodes, so all passed an `UnsupportedChunkSource` to `execute`.